common/src/main/java/gov/cms/ab2d/common/service/CoverageRuntimeException.java

```diff
@@ -0,0 +1,12 @@
+package gov.cms.ab2d.common.service;
+
+public class CoverageRuntimeException extends RuntimeException {
```

Comment:
I did not find anywhere explicit usage of this call (in terms of catching this particular Exception).

It merely wraps RuntimeException and I don't think it pulls its weight.  In other words, I don't see the class providing any value.